src/org/usfirst/frc/team467/robot/Elevator.java

```diff
@@ -14,32 +16,32 @@
 	private static final Logger LOGGER = Logger.getLogger(Elevator.class);
 
 	private WPI_TalonSRX heightController;
-	private int maxTicksPerIteration = 100;
-	private double previousHeight;
 	private MotorSafetyHelper m_safetyHelper;
 
 	private Stops targetHeight;
 
+	private final int ALLOWABLE_ERROR_TICKS = 3;
+	private final int LIMIT_BUFFER = 10;
+
 	public enum Stops {
 		// null if no stop is desired
-		floor(10.0 + RobotMap.ELEVATOR_ERROR_TOLERANCE_INCHES),
-		fieldSwitch(24.0 + RobotMap.ELEVATOR_ERROR_TOLERANCE_INCHES),
-		lowScale(72.0 + RobotMap.ELEVATOR_ERROR_TOLERANCE_INCHES),
-		highScale(96.0 + RobotMap.ELEVATOR_ERROR_TOLERANCE_INCHES);
-
+		basement(761),
```

Comment:
Add a comment describing where these values come from, i.e. these were measured empirically by moving the elevator manually to the desired height and measuring the ticks.

Reply:
Also, I added a new Trello card for us to create an Elevator Calibration Wizard.